groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC
# Auto-generated files were removed with updating gitignore.

**Problem:**
1. Were found four incorrectly generated (and pushed then) files inside the `client` folder.
2. Also we don't wan't to see files generated by `.NET` inside the `bin/obj` folders in the project with tests as the changes.
3. Finally we don't wan't to have `package-lock.json` file under source control.

**Solution:**
1. Files were removed.
2. `.gitignore`-file was updated.
3. File was removed. 

**Tests:**
1. Run build for the solution.
2. Check empty changes after successful build finish.
